bitnami/mariadb-galera/templates/statefulset.yaml

```
@@ -49,6 +50,25 @@ spec:
         - name: "mariadb-galera"
           image: {{ template "mariadb-galera.image" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.galera.bootstrap.forceBootstrap }}
+          command:
+            - bash
+            - -ec
+            - |
+                # Bootstrap from the indicated node
+                HOSTNAME=$(hostname -s)
+                if [[ $HOSTNAME =~ (.*)-([0-9]+)$ ]]; then
+                    NODE=${BASH_REMATCH[2]}
+                    if (( "$NODE" -eq "{{ int .Values.galera.bootstrap.bootstrapFromNode }}" )); then
```

Previous suggestions are not applied:
```suggestion
                HOSTNAME="$(hostname -s)"
                if [[ "$HOSTNAME" =~ "(.*)-([0-9]+)$" ]]; then
                    NODE="${BASH_REMATCH[2]}"
                    if [[ "$NODE" -eq "{{ int .Values.galera.bootstrap.bootstrapFromNode }}" ]]; then
```